macros_doc.md

```diff
@@ -9,15 +34,40 @@ such as Excel.
 
 MkDocs (the tool we use to turn the markdown version of the specification into
```

Comment:
Add BIDS before specification here and remove BIDS from next sentence:

MkDocs (the tool we use to turn the markdown version of the BIDS specification into